apps/web/src/components/forms/montitor-form.tsx

```diff
@@ -43,6 +43,7 @@ export function MonitorForm({ id, defaultValues, onSubmit }: Props) {
       url: defaultValues?.url || "",
       name: defaultValues?.name || "",
       description: defaultValues?.description || "",
+      periodicity: defaultValues?.periodicity || undefined,
```

Comment:
How about adding `"10m"` as default?

Reply:
but if we set 10m by default the combo box will always be 10m and we won't see the empty states for the combo box

Reply:
Fine by me. Thought it would require one less operation for the user to do. We do have the "description" and "label" for explanation.

Reply:
![CleanShot 2023-07-16 at 13 18 15](https://github.com/openstatusHQ/openstatus/assets/13894054/da96c66d-0b76-4787-b158-d0d6411f3fc8)